modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -16,28 +16,38 @@ The *Pipelines* view in the *Developer* perspective lists all the pipelines in a
 [Discrete]
 .Procedure
 . In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down list to see the pipelines in that project.
-. Click the required pipeline to see the *Pipeline details* page. By default, the *Details* tab opens and provides a visual representation of all the serial and parallel tasks in the pipeline. The tasks are also listed in the lower right portion of the page. You can click the listed *Tasks* to view the task details.
+. Click the required pipeline to see the *Pipeline details* page. By default, the *Details* tab opens and provides a visual representation of all the serial tasks, parallel tasks, and the final tasks, along with the `when` expressions, if any, in the pipeline. The tasks and the final tasks are also listed in the lower right portion of the page. You can click the listed *Tasks* and *Finally tasks* to view the task details.
```

Comment:
when expressions

Reply:
Are they seeing the page or is the page opening? 
"The *Details* tab displayed a visual..."
This list is confusing. I would not use the words "along with'. Just make it a running list separated by commas or put it in a bulleted list. 
"the tasks... are listed" no need for also. 
Remove "You can" from that last sentence.